requests-for-comments/2024-08-14-youtuber.md

```diff
@@ -0,0 +1,30 @@
+# Request for comments: Hire a YouTuber
+
+PostHog is built on content + word of mouth. YouTube has proven to be an effective channel for word of mouth grow, based on the significant number of people mention YouTube (or Theo) on signup, but we don’t have the expertise to do content right there. We differentiate by 
+
+YouTube rewards consistency and iteration. Our subscriber base and watch time continues to grow even though we don’t upload more videos. YouTube doesn’t work if you do it part time. You need to go all-in on video. 
+
+I think we have seen some successes, but have been too scattered to really capitalize on them:
+
+- Theo + YouTube mentions shows its promise
+- Although YouTube **tutorials** about PostHog don’t seem to work super well, topic-based (like blogs) YouTube videos do.
+- Although, anecdotally, my small number of YouTube videos get **more** mention than my blogs or newsletters from people, especially when they are learning PostHog. The complexity of PostHog lends itself to video. It is a great way to teach people how to use it.
+
+Beyond YouTube, what could a YouTuber do:
+
+- Create video social content + changelog like so many people have wanted to.
+- Experiment with short form content.
+- Collaborate with influencers (we have done this with newsletters and it works well). This would also help with my influencer plan.
+- Read out 20,000 names (just kidding… unless…).
+
+This does require the **right person**, a proper YouTuber, probably someone who already is one now. I think this is a really cool role for them and PostHog is the right company to attract them. We could talk to the YouTubers we are working with now. 
+
+**What does great look like?** YouTube grows as a source of signups. YouTuber is publishing multiple videos per week. Videos are consistently hitting 10k+ views, some viral hits. Developing video as expertise at PostHog.
+
+**Why not another (writing) content person?** There is always more newsletters, docs, tutorials, and blogs to do, but I don’t feel it is enough to add another content person (beyond filling in for James). 
```

Comment:
Just so I'm clear, you're suggesting we hire this role instead of replacing James, or in addition to?

Reply:
I read this as in addition to replacing James